### DISCUSSION TITLE
Rook suport to wiper device and delete exists dir(/var/lib/rook..) automatically, after delete cluster?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#10134] [Q&A] [answered]
After we deleted the cluster, if we want to re-install the ceph cluster, we need to delete exists dir and wiper the device. It's possible to delete the not needed exists and wiper device automatically.
@travisn

**Comment by travisn** [comment, marked as answer]
@zhucan You can clean the host path and wipe the OSD devices by setting the `cleanupPolicy`. See the [cleanup documentation](https://rook.github.io/docs/rook/v1.9/ceph-teardown.html#delete-the-cephcluster-crd).

**Reply by zhucan** [reply]
So if we want to clean the cluster，we can only run “kubectl patch” with the cleanpolicy？

**Reply by travisn** [reply]
Correct, you need to patch the CephCluster with the cleanup policy, then delete the CephCluster CR.

**Reply by zhucan** [reply]
Thanks
